### ISSUE TITLE
Is this project dead or alive?

### ISSUE THREAD

**Issue by ms-ati** [acmcarther/cucumber-rs#54]
Canary issue: Is this project dead or alive?

If this issue sits here unremarked for a length of time, folks who come later can regard it as dead. If it is discussed we know it's alive! :)

**Comment by acmcarther** [comment]
The project isn't actively maintained, but I'd accept PRs, and provide guidance to get things back together. I can't make any real commitment to ongoing features (or even to get the thing to build again) in the near term though.

I'm not particularly happy with the current implementation though...

**Comment by ms-ati** [comment]
Thanks for your response @acmcarther!

It would be great as a newcomer exploring Rust, where it seems that so many of the highest-rated libraries on crates.io are dead or dying, if this sort of information could be placed front-and-center at the top of the `README.md`.

Would you be willing to declare it "mostly dead" or "looking for new maintainers"? This would be so helpful and reduce the sense of frustration folks discover projects.

**Comment by acmcarther** [comment]
I added a minimal note to the readme in d2e2e75b8230fd0f3aa81667777d0044b31f6d6c

As an unsolicited bit of advice -- I'm currently under the impression that Cucumber is ill suited to Rust or any language with its combination of rigidness around polymorphism and lack of reflection. That said, https://github.com/cucumber/cucumber-cpp seems to be proving me wrong and I'd welcome more torch bearers for the cause of TDD in Rust.

**Comment by ms-ati** [comment]
Thanks so much -- perhaps new maintainers will appear! Perhaps there should be a mailing list just for doing BDD in Rust, and focus on *one* community-supported version of Cucumber / Gherkin, and *one* community-supported version of RSpec? (since reem/stainless is also, quietly, dead)

**Comment by bbqsrc** [comment]
I had a look at getting the Ruby Cucumber test runner to work with Rust again, but I didn't feel like learning Ruby and their documentation is insufficient to fix the problem at hand.

So I rewrote the [gherkin parser in Rust](https://github.com/bbqsrc/gherkin-rust) and the [test runner in Rust](https://github.com/bbqsrc/cucumber-rust) too. I will be maintaining this for a long period of time as I am already using it production and developing it alongside my production needs as they come.

Example of the current output below:

<img width="1234" alt="screen shot 2018-07-03 at 13 24 16" src="https://user-images.githubusercontent.com/279099/42221412-368901cc-7ed2-11e8-838e-2e6a4dc81ecf.png">

@acmcarther if you want to push people in this direction, feel free. Just dropping this here so people won't feel lost when looking for a Cucumber solution for Rust. :smile:

**Comment by acmcarther** [comment]
I was quietly watching your project. It looks great, added a notice at the top of this readme to point toward cucumber-rust: https://github.com/acmcarther/cucumber/commit/510352a1c9861bff506832e1d824eafe2dd5b0b5

**Comment by bbqsrc** [comment]
Cheers! 👍 

Hopefully I'll be able to prove you wrong about TDD-related tooling with intimidatingly strongly typed languages, hehe.

**Comment by bbqsrc** [comment]
Hey @acmcarther, it's been almost a year and [bbqsrc/cucumber-rust](https://github.com/bbqsrc/cucumber-rust) continues to be developed and maintained. I was wondering if it would be possible to adopt the `cucumber` crate from you on crates.io if you would be so kind? :smile:

**Comment by acmcarther** [comment]
@bbqsrc:

I'm open to this, provided that bbqsrc/cucumber-rust has become the official way to use Cucumber -- or rather -- provided that there is an open line of communication between Cucumber maintainers and your crate's maintainers to establish that your crate is the right way to use Cucumber. I say this not as a defense of my old unmaintained crate, just out of curiosity and interest in what the state of Cucumber + Rust is today.

TL;DR: I'm selfishly somewhat disinclined to hand over the name if that crate hasn't become the "official" crate.

**Comment by tyranron** [comment]
@acmcarther hi, the `cucumber-rust` crate is [referred in official Cucumber docs as a Rust tool](https://cucumber.io/docs/tools/related-tools/) for quite a while now. Also, the project has grown, turned into an [organization](https://github.com/cucumber-rs/cucumber) and now has at least 3 maintainers. Wouldn't you mind to reconsider your decision to give us `cucumber` and `gherkin` crates ownership?

**Comment by acmcarther** [comment]
Sent you an invite to own `cucumber`.

[I don't own `gherkin`](https://crates.io/crates/gherkin).

**Comment by acmcarther** [comment]
Feel free to remove me once you've pushed your crate to crates.io.

**Comment by tyranron** [comment]
@acmcarther thanks!
